src/utils/translation.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Supported languages.
+ */
+export type Language = 'en';
```

Comment:
pas sensé être un enum / string[] ca?

Reply:
il est possible de faire un string union type aussi. C'est juste que l'on supporte juste 'en for now'